frontend/src/ui/OnetLink.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export interface OnetLinkProps {
+  socCode: string;
+}
+
+const OnetLink = (props: OnetLinkProps) => (
+  <React.Fragment>
+    <a href={`https://www.mynextmove.org/profile/summary/${props.socCode}`}>
```

Comment:
My understanding is that we're going to use the 6-digit soc code for the app (soc_code in jobs_blsoes), but from your story https://www.mynextmove.org/profile/summary/15-1131 doesn't resolve. So I think we'll need to provide the soc_decimal_code in the table data to render this. 

Separately, have we decided which occupation to use when a particular 6-digit SOC code has more than one decimal code?

Reply:
just to close the loop on this one -- I did some digging and found no cases where there's a mapping of more than one english name for a given code. Also found no cases that did not end with .00. I am updating the code to handle either type of input though.